kedro-telemetry/tests/test_plugin.py

```diff
@@ -433,3 +432,14 @@ def test_after_context_created_with_kedro_run(
 
         # CLI hook makes the first 2 calls, the 3rd one is the Project hook
         assert mocked_heap_call.call_args_list[2] == expected_call
+
+    def test_after_context_created_no_consent_given(self, mocker, fake_context):
```

Comment:
This looks like a good test but just to be sure: does it fail before you applied your fix?